### PR TITLE
[Job Launcher] Added minor improvements

### PR DESCRIPTION
**Pull request by eugenvoronov**
## Description
- Remove IMAGE_LABEL_BINARY type of task
- Add `user_guide` to job creation for CVAT
- Update `event_type`

## How test the changes
`yarn test`

## Related issues
[[Job Launcher] Hide IMAGE_LABEL_BINARY job type#882](https://github.com/humanprotocol/human-protocol/issues/882)

### CODE REVIEW

File: packages/apps/job-launcher/server/src/common/enums/webhook.ts [hunk 1]
```diff
@@ -1,7 +1,10 @@
 export enum EventType {
   ESCROW_CREATED = 'escrow_created',
   ESCROW_CANCELED = 'escrow_canceled',
+  TASK_REJECTED = 'task_rejected',
   TASK_CREATION_FAILED = 'task_creation_failed',
+  TASK_COMPLETED = 'task_completed',
+  TASK_FINISHED = 'task_finished'
```
Comment (Dzeranov) [inline packages/apps/job-launcher/server/src/common/enums/webhook.ts:7]: @eugenvoronov why did you add these events? Not sure we need to know about event types that are actually shared between exchange and recording oracles only.

Comment (eugenvoronov) [reply packages/apps/job-launcher/server/src/common/enums/webhook.ts:7]: @Dzeranov  I have decided to bring this enum to one type in the services where it is used. Basically, if you are sure that none of these events will be sent to the launcher, we can delete them.

**Review by Dzeranov — APPROVED** [review]

**Review by portuu3 — APPROVED** [review]
